src/qiboconnection/api.py

```diff
@@ -767,6 +781,205 @@ def delete_runcard(self, runcard_id: int) -> None:
             raise RemoteExecutionException(message="Runcard could not be removed.", status_code=status_code)
         logger.info("Runcard %i deleted successfully with message: %s", runcard_id, response)
 
+    # CALIBRATIONS
+
+    def _create_calibration_response(self, calibration: Calibration):
+        """Make the calibration create request and parse the response"""
+        response, status_code = self._connection.send_post_auth_remote_api_call(
+            path=self._CALIBRATIONS_CALL_PATH,
+            data=asdict(calibration.calibration_request()),
+        )
+        if status_code not in [200, 201]:
+            raise RemoteExecutionException(message="Calibration could not be saved.", status_code=status_code)
```

Comment:
If there is an error, we should log more info than the status code

Reply:
If the codes are error-like (`4xx`, `5xx`) that should be caught by `send_post_auth_remote_api_call` from the connection module, that passes the http messages as error logs in the client stack.
 Here the will is to have something that restricts to the OK codes we'd expect from our PublicAPI.